src/data_structures/stack.cairo

```diff
@@ -74,7 +74,7 @@ impl Felt252StackImpl<
     /// * Option<u256> The item removed or None if the stack is empty.
     fn pop(ref self: Felt252Stack<T>) -> Option<T> {
         if self.is_empty() {
-            return Option::None(());
+            return Option::None;
```

Comment:
```suggestion
            Option::None;
```

Reply:
![image](https://github.com/keep-starknet-strange/alexandria/assets/16206518/d8422748-1957-41e5-a5c6-a7c973f5309c)

Reply:
indeed, didn't see the end of the function